### ISSUE TITLE
Linux Physx 3.3.2 linker error

### ISSUE THREAD

**Issue by highway900** [taylor001/crown#49]
Hello :)

I cannot seem to get crown to link to Physx...

```
Linking 
/usr/bin/ld: cannot find -lLowLevelCloth  
/usr/bin/ld: cannot find -lPhysX3  
/usr/bin/ld: cannot find -lPhysX3Common  
/usr/bin/ld: cannot find -lLowLevel  
/usr/bin/ld: cannot find -lPhysX3CharacterKinematic  
/usr/bin/ld: cannot find -lPhysX3Cooking  
/usr/bin/ld: cannot find -lPvdRuntime  
/usr/bin/ld: cannot find -lSceneQuery  
/usr/bin/ld: cannot find -lSimulationController
```

Things I have done...

- set the environment variable `PHYSX_SDK_LINUX` to the location I unpacked Physx.
- added the `$PHYSX_SDK_LINUX/Bin/linux64` directory to /etc/ld.conf.d/physx.conf
- ..so `ldconfig -v` checks out with ...
```physx-3.3.2_linux_sdk_core/Bin/linux64:
        libPhysX3CommonCHECKED_x64.so -> libPhysX3CommonCHECKED_x64.so
        libPhysX3PROFILE_x64.so -> libPhysX3PROFILE_x64.so
        libPhysX3CookingPROFILE_x64.so -> libPhysX3CookingPROFILE_x64.so
        libPhysX3CookingDEBUG_x64.so -> libPhysX3CookingDEBUG_x64.so
        libPhysX3CommonDEBUG_x64.so -> libPhysX3CommonDEBUG_x64.so
        libPhysX3_x64.so -> libPhysX3_x64.so
        libPhysX3CharacterKinematicDEBUG_x64.so -> libPhysX3CharacterKinematicDEBUG_x64.so
        libCgGL.so -> libCgGL.so
        libPhysX3GpuPROFILE_x64.so -> libPhysX3GpuPROFILE_x64.so
        libPhysX3GpuDEBUG_x64.so -> libPhysX3GpuDEBUG_x64.so
        libPhysX3Gpu_x64.so -> libPhysX3Gpu_x64.so
        libPhysX3Cooking_x64.so -> libPhysX3Cooking_x64.so
        libPhysX3CharacterKinematicCHECKED_x64.so -> libPhysX3CharacterKinematicCHECKED_x64.so
        libPhysX3GpuCHECKED_x64.so -> libPhysX3GpuCHECKED_x64.so
        libPhysX3CharacterKinematicPROFILE_x64.so -> libPhysX3CharacterKinematicPROFILE_x64.so
        libPhysX3CookingCHECKED_x64.so -> libPhysX3CookingCHECKED_x64.so
        libPhysX3Common_x64.so -> libPhysX3Common_x64.so
        libPhysX3CommonPROFILE_x64.so -> libPhysX3CommonPROFILE_x64.so
        libPhysX3DEBUG_x64.so -> libPhysX3DEBUG_x64.so
        libCg.so -> libCg.so
        libPhysX3CHECKED_x64.so -> libPhysX3CHECKED_x64.so
        libPhysX3CharacterKinematic_x64.so -> libPhysX3CharacterKinematic_x64.so
```
- I even tried moving the `Lib/linux64/*.a` static libs to `/usr/local/lib`

So I am a little confused as to why the linked cannot find the library. Some of these names do not marry up at all such as `-lLowLevel` ... But my C++ is a bit rusty so this could easily be the problem.

Cheers.

**Comment by highway900** [comment]
Ok so I got everything to compile, but it is not very pretty.... I had to remove linked libraries in the genie script
```
		configuration { "release", "linux-*" }
			linkoptions {
				"-Wl,--start-group $(addprefix -l," ..
				--"	LowLevel" ..
				--"	LowLevelCloth" ..
				"	PhysX3_x64" ..
				"	PhysX3Common_x64" ..
				"	PxTask" ..
				"	PhysX3CharacterKinematic_x64" ..
				"	PhysX3Cooking_x64" ..
				"	PhysX3Extensions" ..
				"	PhysX3Vehicle" ..
				"	PhysXProfileSDK" ..
				"	PhysXVisualDebuggerSDK" ..
				--"	PvdRuntime" ..
				--"	SceneQuery" ..
				--"	SimulationController" ..
				") -Wl,--end-group"
			}
```
This is obviously going to cause me issues later on. But I do not seem to have these library files. 

Have I potentially not built Physx correctly? (though my understanding is the precompiled binaries should be enough)

**Comment by underdoeg** [comment]
I have the same issue. Did you find anything?

**Comment by taylor001** [comment]
Fixed in master: efba41315e9dc3d1a592a266ff0e243438f88712

**Comment by underdoeg** [comment]
jep, it works
